template/template.js

```diff
@@ -189,7 +193,8 @@ module.exports = templateSource => {
         const out = apply(raw[Symbol.asyncIterator](), lang, initialJsState);
         write.on("close", resolve);
         write.on("error", reject);
-        out.on("error", write.destroy);
+        // out.on("error", write.destroy) doesn't properly forward the error!
+        out.on("error", err => write.destroy(err));
```

Comment:
I have no idea why this is required. Maybe `destroy` is altered once you start the pipe or something.